packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -182,6 +185,13 @@ export const Popup: React.FC<PopupProps> &
   const triggerFocusableRef = React.useRef<HTMLElement>();
   const rightClickReferenceObject = React.useRef<PopperJs.VirtualElement | null>();
 
+  useFrameFocus(open, context.target, (e: Event) => {
```

Comment:
What I think it's sad here is that even users that has no need for it will have that set interval set when using popup, we should probably no rely only in the `open` flag but also probably pass a flag set in the `Provider` and accessed through `useFluentContext` to enable this feature. WDYT @ling1726 @layershifter ?

Reply:
I don't really see a problem with a `setInterval` call as long as it is only used when necessary (i.e. only when popups are open) and then disposed correctly when they are no longer needed. From a perf perspective no reason is obv for me here, maybe @layershifter will have some.

I think a context configuration is a good idea, so that it's possible to configure the iframe behaviour globally for a subtree, perhaps some experiences in Teams do not need this behaviour.

Reply:
It bothers me to think as a lib that anyone who is going to use Popup/Menu/Tooltip is going to have this unintentional

Reply:
> I don't really see a problem with a `setInterval` call as long as it is only used when necessary (i.e. only when popups are open) and then disposed correctly when they are no longer needed. From a perf perspective no reason is obv for me here, maybe @layershifter will have some.

I don't see reasons why we should hide this under feature/perf flag as if it's affecting perf we will need to make it performant anyway 😅